src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('span.population');
+
+const populationValues = [...populations].map(value => (
+  +value.innerText.split(',').join('')
+));
+const totalPopulation = populationValues.reduce((prev, value) =>
+  prev + value, 0);
+const averagePopulation = totalPopulation / populationValues.length;
+
+document.querySelector('.total-population')
+  .innerText = totalPopulation.toLocaleString();
+
+document.querySelector('.average-population')
+  .innerText = averagePopulation.toLocaleString();
```

Comment:
[textContent](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations) is safer